mds/access_control/authenticate.py

```diff
@@ -38,7 +39,11 @@ def is_authenticated(self):
 
     @property
     def is_staff(self):
-        return False
+        return 'admin' in self.scopes
```

Comment:
Shouldn't it be named `staff` to keep consistency?

Reply:
Current mds app scopes are : "vehicle", "admin", "prv_api", "provider".
I may have to rename the function by "is_admin" actualy.